pyblish_qml/ipc/server.py

```diff
@@ -104,11 +104,12 @@ def show(self, settings=None):
             settings (optional, dict): Client settings
 
         """
-        if not self.ninja:
+        if self.ninja:
+            return self._dispatch("show", args=[settings or {},
+                                                self._winId,
+                                                self.ninja])
+        else:
             self.vessel.show()
-        return self._dispatch("show", args=[settings or {},
-                                            self._winId,
-                                            self.ninja])
 
```

Comment:
This spot, is the reason that make Maya 2015 and 16 freeze at startup. 
`ninja` is disabled if host running Qt4, and if `ninja` is disabled, it need to call `vessel.show` before `_dispatch("show")`, or the window will not show up.